pilot/helpers/files.py

```diff
@@ -145,10 +145,18 @@ def clear_directory(directory: str, ignore: Optional[list[str]] = None):
             if matcher.ignore(full_path):
                 continue
 
-            os.remove(full_path)
+            try:
```

Comment:
Also catches some weird edge cases. If we can't remove a file here, oh well, not such a big deal, we don't want to crash over it.